DSharpPlus/Entities/Guild/DiscordGuild.cs

```diff
@@ -63,7 +64,7 @@ public class DiscordGuild : SnowflakeObject, IEquatable<DiscordGuild>
         /// </summary>
         [JsonIgnore]
         public string IconUrl
-            => !string.IsNullOrWhiteSpace(this.IconHash) ? $"https://cdn.discordapp.com/icons/{this.Id.ToString(CultureInfo.InvariantCulture)}/{this.IconHash}.jpg" : null;
+            => !string.IsNullOrWhiteSpace(this.IconHash) ? $"https://cdn.discordapp.com/icons/{this.Id.ToString(CultureInfo.InvariantCulture)}/{this.IconHash}.{(this.IconHash.StartsWith("a_") ? "gif" : "png")}?size=1024" : null;
```

Comment:
Don't duplicate code, call newly created method instead.
```suggestion
            => this.GetIconUrl(ImageFormat.Auto, 1024);
```